<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>441</width>
      <height>341</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>340</y>
      <width>441</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>561</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件（&amp;F）</string>
    </property>
    <addaction name="openaction"/>
    <addaction name="saveaction"/>
    <addaction name="separator"/>
    <addaction name="exitaction"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑（&amp;E）</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助（&amp;H）</string>
    </property>
    <addaction name="aboutSoftWareaction"/>
    <addaction name="aboutWebaction"/>
    <addaction name="aboutQtaction"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>打印（&amp;P）</string>
    </property>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_E"/>
   <addaction name="menu"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget_2">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QToolBox" name="toolBox">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="page">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>69</width>
          <height>367</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Page 1</string>
        </attribute>
        <widget class="QPushButton" name="pushButton">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>71</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>30</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>69</width>
          <height>367</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Page 2</string>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="openaction">
   <property name="icon">
    <iconset>
     <normaloff>images/add.png</normaloff>images/add.png</iconset>
   </property>
   <property name="text">
    <string>打开（&amp;O）</string>
   </property>
  </action>
  <action name="saveaction">
   <property name="icon">
    <iconset>
     <normaloff>images/favorite.png</normaloff>images/favorite.png</iconset>
   </property>
   <property name="text">
    <string>保存（&amp;S）</string>
   </property>
  </action>
  <action name="exitaction">
   <property name="icon">
    <iconset>
     <normaloff>images/close.png</normaloff>images/close.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="aboutSoftWareaction">
   <property name="icon">
    <iconset>
     <normaloff>images/link.png</normaloff>images/link.png</iconset>
   </property>
   <property name="text">
    <string>关于软件(&amp;S)</string>
   </property>
  </action>
  <action name="aboutWebaction">
   <property name="icon">
    <iconset>
     <normaloff>images/earth.png</normaloff>images/earth.png</iconset>
   </property>
   <property name="text">
    <string>关于官网(&amp;W)</string>
   </property>
  </action>
  <action name="aboutQtaction">
   <property name="icon">
    <iconset>
     <normaloff>images/timg.gif</normaloff>images/timg.gif</iconset>
   </property>
   <property name="text">
    <string>关于Qt(&amp;Q)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
